server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -134,6 +134,7 @@
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
 import org.apache.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
```

Comment:
This is very convenient, but should we allow jetbrains specific annotations? cc @rohityadavcloud @GutoVeronezi @weizhouapache @wido

Reply:
I am ok with it

Reply:
I am OK with it as well; the project [JetBrains/java-annotations](https://github.com/JetBrains/java-annotations) is highly used, receives upgrades periodically, and the annotations are well documented.